tests/unit/test_is_close.py

```diff
@@ -44,26 +48,26 @@ def test_010(json_diff: Path) -> None:
 
     alpha_pydict: PyDictResult = {
         "nside-8": {
-            "columns": ["index", "llh", "E_in", "E_tot"],
+            "columns": ["index", "llh", "E_in", "E_tot"] + position_time_columns,
             "metadata": {"nside": 8},
             "data": [
-                [0, 496.5, 4643.5, 4736.5],
+                [0, 496.5, 4643.5, 4736.5] + dummy_position_time,
```

Comment:
Is the idea that position and time values should not be included in the comparison logic? If so, then I recommend instead we update the logic at https://github.com/icecube/skyreader/blob/feature-write-position-and-time/skyreader/result.py#L187-L192 to ignore certain fields. This will also help with backward compatibility (ie, comparing previous results).

Reply:
I'd support changing the logic so that if a field is not in the rtol dict argument, that value is ignored. Then use `==` only for `"index"` values.

Currently, if it's not included, then a `==` is used. Also see: https://github.com/icecube/skyreader/blob/feature-write-position-and-time/skyreader/result.py#L32-L37

@tianluyuan what are your thoughts on this?

Reply:
I think it's probably better to update https://github.com/icecube/skyreader/blob/feature-write-position-and-time/skyreader/result.py#L32-L37 so that comparison is performed across the full row? This makes it aligned with `__eq__`, though it would break backwards compatibility.

If we wanted to preserve backwards compatibility then making `rtol_per_field` version dependent could work?

Reply:
Thinking a bit this would require some versioning information to be saved in the result file. If none exist then default to the original fields.

Reply:
> comparison is performed across the full row
works for me!